pkg/cmd/cli/secrets/dockercfg.go

```diff
@@ -37,9 +37,9 @@ If you don't already have a .dockercfg file, you can create a dockercfg secret d
 
   $ %s SECRET_NAME --docker-server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL
 
-If you do already have a .dockercfg file, you can create a dockercfg secret by using:
+If you do already have a .dockercfg file, you can create a dockercfg secret by using the 'oc secrets new' command:
 
-  $ %s SECRET_NAME path/to/.dockercfg
+  $ oc secrets new SECRET_NAME path/to/.dockercfg
```

Comment:
@mfojtik is right, we can't hardcode command names, but `%s` will give `oc secrets new-dockercfg`, which is not the example we want... we need another token for the recommended `oc secrets new` command string